community/kernel/src/main/java/org/neo4j/graphdb/factory/GraphDatabaseSettings.java

```diff
@@ -990,4 +999,30 @@ public BoltConnector( String key )
             DISABLED
         }
     }
+
+    public static BaseSetting<String> prefixSetting( final String name, final Function<String,String> parser, final String defaultValue )
```

Comment:
Could you move this somewhere else? Perhaps org.neo4j.kernel.configuration.Settings?